### PR TITLE
Ut for getPrice, addSubSection and TestModeEnabled

### PR DESCRIPTION
**Pull request by garimaprachi**

### CODE REVIEW

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 1]
```diff
@@ -84,4 +84,54 @@ public function testgetCustomisation()
...
+    {
+        global $current_section;
+        $current_section = 'affordability-widget';
+
+        ob_start();
+        addSubSection();
+        $result = ob_get_contents();
+        ob_end_clean();
+
+        $pluginSubSection = '<li><a href="'.admin_url('admin.php?page=wc-settings&tab=checkout&section=razorpay').'" class="">Plugin Settings</a> | </li>';
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:116]: @garimaprachi leave white space before and after . operator

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 2]
```diff
@@ -84,4 +84,54 @@ public function testgetCustomisation()
...
+        
+        $affordabilitywidgetSubSection = '<li><a href="'.admin_url('admin.php?page=wc-settings&tab=checkout&section=affordability-widget').'" class="current">Affordability Widget</a>  </li>';
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:118]: @garimaprachi leave white space before and after . operator

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 3]
```diff
@@ -84,4 +84,54 @@ public function testgetCustomisation()
...
+        $this->assertStringContainsString($affordabilitywidgetSubSection, $result);
+
+        $this->assertStringContainsString(admin_url(), $result);
+
+        $this->assertStringContainsString('</ul><br class="clear" />', $result);
+    }
+
+    public function testisAffordabilityWidgetTestModeEnabled()
+    {
+        $this->assertSame(false, isAffordabilityWidgetTestModeEnabled());
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:131]: @garimaprachi please check if we can use assertFalse here

File: tests/phpunit/tests/test-affordabilitywidget.php [hunk 4]
```diff
@@ -140,4 +142,54 @@ public function testgetKeyID()
         
         delete_option('woocommerce_razorpay_settings');
     }
-}
+    
+     public function testgetPrice()
+    {
```
Comment (abdulwahidsharief) [inline tests/phpunit/tests/test-affordabilitywidget.php:147]: @garimaprachi fix indentation

Comment (garimaprachi) [reply tests/phpunit/tests/test-affordabilitywidget.php:147]: Fixed

**Review by abdulwahidsharief — APPROVED** [review]
LGTM

**Pull request merged**